examples/prismic/app/blog/page.tsx

```diff
@@ -0,0 +1,29 @@
+import { notFound } from 'next/navigation'
+
+import { MakeswiftComponent } from '@makeswift/runtime/next'
+import { getSiteVersion } from '@makeswift/runtime/next/server'
+
+import { BLOG_EMBEDDED_COMPONENT_ID } from '@/components/Builtins/Blog/Blog.makeswift'
+import { client as MakeswiftClient } from '@/lib/makeswift/client'
+import { getAllBlogs } from '@/lib/prismic/fetchers'
+import { PrismicProvider } from '@/lib/prismic/provider'
+
+export default async function Page() {
+  const blogs = await getAllBlogs()
+
+  const componentSnapshot = await MakeswiftClient.getComponentSnapshot('blog-component', {
```

Comment:
🍹 The ID `blog-component` is a bit confusing as this is an _element_, not a component. Maybe we use `blog-index`?